src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/document-[document]/attributeItem.svelte

```diff
@@ -0,0 +1,86 @@
+<script lang="ts">
+    import { Button, FormList } from '$lib/elements/forms';
+    import { capitalize } from '$lib/helpers/string';
+    import type { Attributes } from '../store';
+    import Attribute from './attribute.svelte';
+
+    export let attribute: Attributes;
+    export let formValues: object = {};
+    export let label: string;
+    export let editing = false;
+
+    function removeArrayItem(key: string, index: number) {
+        formValues = {
+            ...formValues,
+            [key]: formValues[key].filter((_, i) => i !== index)
+        };
+    }
+
+    function addArrayItem(key: string) {
+        formValues = {
+            ...formValues,
+            [key]: [...formValues[key], null]
+        };
+    }
+
+    function getAttributeType(attribute: Attributes) {
+        if ('format' in attribute && attribute.format === 'enum') return 'Enum';
+        return `${capitalize(attribute.type)}${attribute.array ? '[]' : ''}`;
```

Comment:
why is enum the exception here?

Reply:
@TGlide do you remember why we did it like this?

Reply:
But so are ip, url and email... why do we only check for enum?